packages/@react-aria/checkbox/src/utils.ts

```diff
@@ -13,9 +13,9 @@
 import {CheckboxGroupState} from '@react-stately/checkbox';
 
 interface CheckboxGroupData {
-  name: string,
-  descriptionId: string,
-  errorMessageId: string,
+  name?: string,
+  descriptionId?: string,
+  errorMessageId?: string,
```

Comment:
Changed props to optional since undefined was sometimes set to props originally.

ref: https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/checkbox/src/useCheckboxGroup.ts#L51-L53